pkg/agent/interfacestore/types.go

```diff
@@ -33,6 +34,10 @@ const (
 
 type InterfaceType uint8
 
+func (t InterfaceType) String() string {
+	return strconv.Itoa(int(t))
```

Comment:
is this ever used for user-facing logs, or just for the cache index? If it is used for user-facing logs, maybe we could have a switch statement and return a more user-friendly string?